.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Report a bug or a performance issue
+about: Use this template to report unexpected behavior
+title: ''
+labels: ''
+assignees: ''
+---
+
+# Summary
+Provide a short summary of the issue. Sections below provide guidance on what
+factors are considered important to reproduce an issue.
+
+# Version
+Report oneMKL version and githash.
+
+# Environment
+oneMKL works with multiple HW and backend libraries and also depending on the compiler and build environment. Include
```

Comment:
may be "also depends" ? 
Not sure what did you mean by "build environment"?

Reply:
Just "depends" works too.

Reply:
fixed